service/app/controllers/TelemetryController.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * 
+ */
+package controllers;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import controller.mapper.RequestMapper;
+import play.libs.F;
+import play.libs.F.Promise;
+import play.mvc.Result;
+
+/**
+ * This controller will handle all sunbird telemetry request data.
+ * 
+ * @author Manzarul
+ *
+ */
+public class TelemetryController extends BaseController {
+
+	/**
+	 * This method will receive the telemetry data and send it to EKStep to process
+	 * it.
+	 * 
+	 * @return F.Promise<Result>
+	 */
+	public F.Promise<Result> save() {
+		try {
+			String contentTypeHeader = request().getHeader("content-type");

```

Comment:
Use constants for:
content-type
accept-encoding
content-type
dispatchTelemetry